app/connectors/FinancialDetailsConnector.scala

```diff
@@ -191,6 +191,45 @@ class FinancialDetailsConnector @Inject()(
       }
   }
 
+  def getFinancialDetails(taxYearFrom: TaxYear, taxYearTo: TaxYear, nino: String)
```

Comment:
We might need to introduce a new type to represent range of tax years: say something like TaxYearsRange ? or similar not sure if we do have something at the moment in our domain but worth to add it (in the scope of this PR I think this would be optional)

Reply:
next tech story created: https://jira.tools.tax.service.gov.uk/browse/MISUV-9063